src/main/java/com/google/api/codegen/PageStreamingConfig.java

```diff
@@ -94,7 +94,7 @@ public static PageStreamingConfig createPageStreaming(
               resourcesFieldName));
     }
 
-    if (requestTokenField == null || responseTokenField == null || resourcesField == null) {
+    if (requestTokenField == null && responseTokenField == null && resourcesField == null) {
```

Comment:
Ah, right. Updated the condition.

I wanted to allow token fields as optional for grpc-streaming methods (and resources field is still required), so that condition wasn't correct.

The reason for optional token fields is that some grpc streaming methods are paged but no token fields exist (see https://github.com/googleapis/googleapis/blob/62ebd10737b35065a6995cfd25eeca81f1433f0f/google/genomics/v1/reads.proto#L458 for example).

Similarly but not quite same -- it's not apparent how to represent the page tokens for Bigtable's ReadRows request/responses (see https://github.com/googleapis/googleapis/blob/72ebf10ad5b2355df038e275a1543418ea8ac109/google/bigtable/v2/bigtable.proto#L96 but I can explain the details).